nixos/modules/flyingcircus/platform/default.nix

```diff
@@ -167,6 +167,12 @@ in
 
     environment.pathsToLink = [ "/include" ];
     environment.shellInit = ''
+     # Grant easy access to the machine's ENC data for some variables to
+     # shell scripts.
+     export FCIO_LOCATION="${cfg.enc.parameters.location}"
+     export FCIO_ENVIRONMENT="${cfg.enc.parameters.environment}"
+     export FCIO_HOSTNAME="${cfg.enc.name}"
+
```

Comment:
Result from discussion: extract really important bits in a way that is easier to process elsewhere. JSON parsing is hard e.g. from shell scripts - and enc.json has tight permissions, so that non-root processes aren't even able to read it.

We assume that the FCIO_ namespace will be extend in the future.